tasks/fix-cat2.yml

```diff
@@ -1224,6 +1231,7 @@
 
 - name: "MEDIUM | RHEL-07-040180 | PATCH | The operating system must implement cryptography to protect the integrity of Lightweight Directory Access Protocol (LDAP) authentication communications."
   command: "true"
+  when: '"yes" in "{{ rhel_07_040180_audit.stdout_lines }}"'
```

Comment:
Tried using that line originally, but would skip due to conditional checking every time. I ended up wrapping it with braces and quotations because that was the only way I could get it to work properly at the time. 

This is on 2.1.0.